### PR TITLE
Feature/translate strings cx 1608

### PR DESCRIPTION
**Pull request by stefaniacardenas**
The aim of this PR is to receive feedbacks for the translation of the SDK into Spanish. This PR will be merged into PR #294. It is currently being compared against `development` to facilitate review.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        title: 'Página con la foto del pasaporte',
+        instructions: 'Cargue la página con la foto del pasaporte desde su computadora',
+        webcam: 'Coloque la página con la foto de su pasaporte en el cuadro (será detectado automáticamente)',
+      },
+    },
+    document: {
+      help: '¿Tiene algún problema? Cargue un archivo en su lugar',
+    },
+    face: {
+      title: 'Tòmese una selfie',
```
Comment (guiman) [inline locales/es.js]: [minor] I think this has the tilde (acento) on the wrong angle (ò instead of ó)

File: locales/es.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      title: 'Compruebe la legibilidad',
+    },
+    driving_licence: {
+      message: 'Asegúrese de que los datos de su licencia se puedan leer claramente, sin borrosidades ni brillo',
+    },
+    national_identity_card: {
+      message: 'Asegúrese de que los datos de su tarjeta se puedan leer claramente, sin borrosidades ni brillo',
+    },
+    passport: {
+      message: 'Asegúrese de que los datos de su paraporte se puedan leer claramente, sin borrosidades ni brillo',
```
Comment (guiman) [inline locales/es.js]: Small typo `paraporte` -> `pasaporte`

File: locales/es.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    },
+    face: {
+      title: 'Verificar selfie',
+      message: 'Asegúrese de que selfie muestre claramente su cara',
+    },
+    confirm: 'Confirmar',
+    continue: 'Continuar',
+    redo: 'Hágalo otra vez',
```
Comment (guiman) [inline locales/es.js]: Maybe `Inténtelo nuevamente`?

File: locales/es.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
+export const es = {
+  welcome: {
+    title: 'Abra su nueva cuenta bancaria',
+    description_p_1: 'Para abrir una cuenta bancaria, necesitaremos verificar su identidad.',
+    description_p_2: 'Solo tomará un par de minutos.',
```
Comment (ranierorusso) [inline locales/es.js]: Some older users might think this is a typo: http://www.tallerdeletras.com/la-rae-sola-contra-el-acento-del-adverbio-solo/ (I thought it should have been Sólo).

File: locales/es.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    next_button: 'Verificar identidad',
+  },
+  document_selector: {
+    title: 'Verifique su identidad',
+    hint: 'Seleccione el tipo de documento que desea cargar',
```
Comment (ranierorusso) [inline locales/es.js]: I'd mostly heard 'subir' as the verb for uploading. Not sure which is more common though.

File: locales/es.js (hunk 4 above)
Comment (blancaag) [inline locales/es.js]: 'Solo tomará  = 'Solo nos llevará'

File: locales/es.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  },
+  capture: {
+    driving_licence: {
+      front: {
+        title: 'Frente de la licencia',
+        instructions: 'Cargue el frente de la licencia desde su computadora',
```
Comment (ranierorusso) [inline locales/es.js]: Again, computadora is more a south american term. Ordenador is the Spanish term. What audience are we aiming for?

File: locales/es.js [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    },
+    passport: {
+      front: {
+        title: 'Página con la foto del pasaporte',
+        instructions: 'Cargue la página con la foto del pasaporte desde su computadora',
+        webcam: 'Coloque la página con la foto de su pasaporte en el cuadro (será detectado automáticamente)',
+      },
+    },
+    document: {
+      help: '¿Tiene algún problema? Cargue un archivo en su lugar',
```
Comment (ranierorusso) [inline locales/es.js]: Maybe 'De modo alternativo, también puede cargar/subir un archivo.'

File: locales/es.js (hunk 6 above)
Comment (stefaniacardenas) [reply locales/es.js]: We are aiming to a South American/Mexican audience.

File: locales/es.js (hunk 1 above)
Comment (stefaniacardenas) [reply locales/es.js]: Our audience is from South America/Mexico. Apparently in Mexico selfie is feminine.  🤔

File: locales/es.js [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+    redo: 'Hágalo otra vez',
+  },
+  capture_parentheses: '(o arrastre aquí su archivo )',
+  complete: {
+    message: 'Verificación completa',
+    submessage: 'Gracias.'
+  },
+  cross_device: {
+    client_success: {
+      title: 'Cargas exitosas',
```
[the remark this answers is not in the document]
Comment (stefaniacardenas) [reply locales/es.js]: That would be a bit long for a title. Would "Actualizaciones satisfactorias" work?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+      action: 'Enviar verificación',
+      multiple_docs_uploaded:'Documentos cargados',
+      one_doc_uploaded: 'Documento cargado',
+    },
+    phone_number_placeholder: 'Ingrese el número de celular',
+    loading: 'Cargando...',
+    mobile_connected: {
+      title: {
+        message: 'Conectado a su móvil',
+        submessage: 'Una vez que haya terminado, lo llevaremos al próximo paso',
```
Comment (ranierorusso) [inline locales/es.js]: 'le' would be gender neutral i think.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+    switch_device: {
+      header: '¿Necesita usar su móvil para tomar fotos?',
+      submessage: 'Continúe la verificación de forma segura en su dispositivo móvil',
+    },
+    tips: 'Tips',
+  },
+  errors: {
+    invalid_capture: {message:'No se detectó ningún documento', instruction: 'Asegúrese de que todo el documento esté en la foto'},
+    invalid_type: {message: 'Archivo no cargado', instruction: 'Intenta usar otro tipo de archivo'},
+    unsupported_file: {message: 'Tipo de archivo no soportado', instruction: 'Intente usar un archivo .jpg o .png'},
```
Comment (ranierorusso) [inline locales/es.js]: Maybe 'Tipo de archivo no admitido' or 'no compatible'. Soportado makes me think you can't stand it haha. I could be wrong though.

File: locales/es.js [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+    invalid_size: {message: 'Tamaño de archivo demasiado grande', instruction: 'El tamaño debe ser menor de 10 MB'},
+    no_face: {message: 'Rostro no encontrado', instruction: 'Su cara debe de estar en la selfie'},
```
Comment (ranierorusso) [inline locales/es.js]: Why Rostro here but Cara everywhere else?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+    multiple_faces: {message: 'Múltiples caras encontradas', instruction: 'Solo su cara puede estar en la selfie'},
+    server_error: {message: 'Conexión perdida', instruction: 'Inténtalo de nuevo'},
+    glare_detected: {message: 'Detectado brillo', instruction: 'Todos los detalles deben ser claros y legibles'},
```
Comment (ranierorusso) [inline locales/es.js]: Isn't it Brillo detectado? I think you might need a 2nd opinion for this one though.

File: locales/es.js [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+    sms_failed: {message: 'Algo salió mal', instruction: "Copie el enlace a continuación en su dispositivo móvil"},
+    sms_overuse: {message: 'Demasiados intentos de reenvío', instruction: "Copie el enlace a continuación en su dispositivo móvil"},
+    lazy_loading: {message: 'Se produjo un error al cargar el componente'},
+    invalid_number: {message: 'Comprueba que su número de móvil sea correcto'},
```
Comment (ranierorusso) [inline locales/es.js]: I think you might be mixing tenses and you'd want Compruebe here.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+    generic_client_error: {message: 'Algo salió mal', instruction: 'Deberá reiniciar tu verificación en su computadora'},
+  },
+  passport: 'Pasaporte',
+  driving_licence: 'Licencia de conducir',
+  national_identity_card: 'Tarjeta de identificación',
+  short_passport: 'Pasaporte',
```
Comment (ranierorusso) [inline locales/es.js]: the other short_ ones are lower case.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js (hunk 14 above)
Comment (stefaniacardenas) [reply locales/es.js]: Good point, they are supposed to be lower case.

File: locales/es.js [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+  short_driving_licence: 'licencia',
+  short_national_identity_card: 'tarjeta',
+  back: 'atrás',
+  cancel: 'Cancelar',
```
[the remark this answers is not in the document]
Comment (stefaniacardenas) [reply locales/es.js]: No, `back` and `cancel` are correct

File: locales/es.js (hunk 12 above)
Comment (stefaniacardenas) [reply locales/es.js]: I think they would both work but, you're right, "Brillo detectado" sounds better.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://295-pr-onfido-sdk-ui-onfido.surge.sh

File: locales/es.js [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+    submessage: 'Gracias.'
+  },
+  cross_device: {
+    client_success: {
+      title: 'Cargas exitosas',
+      sub_title: 'Ahora puede regresar a su computadora para continuar',
+    },
+    link: {
+      title: 'Continúe la verificación en su dispositivo móvil',
+      sub_title: 'Le mandaremos un mensaje de texto único con un enlace seguro a su móvil',
```
[the remark this answers is not in the document]
Comment (stefaniacardenas) [reply locales/es.js]: For legal reasons we have to specify that the text will only be sent once.  The original string is 'We’ll text a one-time secure link to your mobile'. Would "único" convey that meaning?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-295-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://295-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request closed without merging**